osu.Game/Database/SingletonContextFactory.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) 2007-2018 ppy Pty Ltd <contact@ppy.sh>.
+// Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu/master/LICENCE
+
+namespace osu.Game.Database
+{
+    public class SingletonContextFactory : IDatabaseContextFactory

```

Comment:
It doesn't look like this class is ever used.

Reply:
`TestCasePlaySongSelect`